recipes/mash/2.3/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.3" %}
+
+package:
+  name: mash
+  version: "{{ version }}"
+
+about:
+  home: http://mash.readthedocs.org
+  license: Other
+  summary: A fast sequence distance estimator using MinHash.
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/marbl/Mash
+  git_rev: v{{ version }}
+  patches:
+    - Makefile.in.patch
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - autoconf
+    - automake
+    - libtool
+    - make
+  host:
+    - boost
+    - capnproto
+    - libz-dev
+  run:
+    - boost
+    - capnproto
```

Comment:
Please check that this is required at runtime and if so, split the capnproto recipe into two sub-packages (one with the components required to build and one with the runtime components).